modules/run-vault/run-vault

```diff
@@ -14,10 +14,12 @@ readonly DEFAULT_AGENT_AUTH_MOUNT_PATH="auth/aws"
 readonly DEFAULT_PORT=8200
 readonly DEFAULT_LOG_LEVEL="info"
 
+readonly DEFAULT_CONSUL_AGENT_SERVICE_REGISTRATION_ADDRESS="localhost:8500"
+
 readonly EC2_INSTANCE_METADATA_URL="http://169.254.169.254/latest/meta-data"
 
 readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-readonly SCRIPT_NAME="$(basename "$0")"
+readonly SCRIPT_NAME="$(basename "${0}")"
```

Comment:
Why all the variable format changes? We don't usually do that when not necessary. Unless intentional, could you revert these? They make this diff very large.

Reply:
I wanted to have the same format for all variables because there was a mix of usage:
```bash
  local -r s3_bucket="$9"	  
  local -r s3_bucket_path="${10}"
```
I'll revert anyway